weechataboo.scm

```diff
@@ -75,25 +75,28 @@
   (let*
     ; Some defaults which may be useful‥
     ((emotions
-       '(("angry" "눈_눈,(¬_¬),(｀ε´),(¬▂¬),（▽д▽）")
```

Comment:
Could you please indent all emotion-strings to the depth of the longest category name (*aggressive* in this case) and resort the list alphabetically based on the categories.

```scheme
; Like this
("aggressive" "…")
("angry"      "…")
("blush"      "…")
```

Just saw that I already had a wrong indenting with *excited*. This makes this PR even more important :^)